src/devices/acurite.c

```diff
@@ -23,14 +23,18 @@
 #define ACURITE_TXR_BITLEN        56
 #define ACURITE_5N1_BITLEN        64
 #define ACURITE_6045_BITLEN        72
+#define ACURITE_ATLAS_BITLEN      80
 
 // ** Acurite known message types
 #define ACURITE_MSGTYPE_TOWER_SENSOR                    0x04
 #define ACURITE_MSGTYPE_6045M                           0x2f
 #define ACURITE_MSGTYPE_5N1_WINDSPEED_WINDDIR_RAINFALL  0x31
 #define ACURITE_MSGTYPE_5N1_WINDSPEED_TEMP_HUMIDITY     0x38
-#define ACURITE_MSGTYPE_WINDSPEED_TEMP_HUMIDITY_3N1     0x20
+#define ACURITE_MSGTYPE_3N1_WINDSPEED_TEMP_HUMIDITY     0x20
 #define ACURITE_MSGTYPE_RAINFALL                        0x30
+#define ACURITE_MSGTYPE_ATLAS_WINDSPEED_TEMP_HUMIDITY   0x25
```

Comment:
Does the message type change depending on whether the lightning detector is installed?

Reply:
Yes, this changes. 

LD     |  No LD
0x25  |  0x05
0x26  |  0x06
0x27  |  0x07